c7n/policy.py

```diff
@@ -802,6 +802,34 @@ def validate(self):
                 'policy:%s resource:%s does not have a cloudformation type'
                 ' and is there-fore not supported by config-poll-rule'))
 
+    @staticmethod
+    def get_obsolete_evaluations(client, cfg_rule_name, ordering_ts, evaluations):
+        """Get list of evaluations that are no longer applicable due to resources being deleted
+        """
+        latest_resource_ids = set()
+        for latest_eval in evaluations:
+            latest_resource_ids.add(latest_eval['ComplianceResourceId'])
+
+        obsolete_evaluations = []
+        paginator = client.get_paginator('get_compliance_details_by_config_rule')
```

Comment:
import and add in, `paginator.PAGE_ITERATOR_CLS = RetryPageIterator`

Reply:
Imported and added RetryPageIterator